composer.json

```diff
@@ -41,11 +41,13 @@
         }
     },
     "require": {
-        "php": ">=5.3"
+        "php": ">=5.3",
+        "ext-iconv": "*",
```

Comment:
Is there any reason to require those two PHP extensions? Where? Punic shouldn't require them (one of the key points of Punic is that it should work in any environment, the only requirement is PHP 5.3)

Reply:
See this [result](https://github.com/punic/punic/search?utf8=%E2%9C%93&q=function_exists&type=) and some places that is possible to use them.

Reply:
Punic uses mbstring/iconv *if* they are available, it doesn't require them...

Reply:
How about using the ```suggest``` in ```composer.json```?

Do you have the proper description?